staging/src/k8s.io/kubectl/pkg/cmd/label/label.go

```diff
@@ -187,11 +187,16 @@ func (o *LabelOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []st
 	}
 	o.dryRunVerifier = resource.NewQueryParamVerifier(dynamicClient, f.OpenAPIGetter(), resource.QueryParamDryRun)
 
-	o.ToPrinter = func(operation string) (printers.ResourcePrinter, error) {
-		o.PrintFlags.NamePrintFlags.Operation = operation
-		// PrintFlagsWithDryRunStrategy must be done after NamePrintFlags.Operation is set
```

Comment:
I'd like ask @brianpursley  to assure that this change is ok, since he added this comment.

Reply:
I should have said *why* in the comment, but it was to fix this issue (https://github.com/kubernetes/kubernetes/pull/111571) where it was not indicating that it is a dry run in the output, making it seem like it really applied your label change.

I added unit tests during that PR, so if the unit tests still pass, then it should be good.